pom.xml

```diff
@@ -40,7 +40,7 @@
         <dependency>
             <groupId>com.github.ONSdigital</groupId>
             <artifactId>dp-dd-backend-model</artifactId>
-            <version>develop-SNAPSHOT</version>
+            <version>feature~metadata-SNAPSHOT</version>
```

Comment:
If these commits are merged at the same time, shouldn't this remain as develop-SNAPSHOT?